dts/bindings/ethernet/nxp,enet-mac.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP ENET MAC/L2 Device
+
+compatible: "nxp,enet-mac"
+
+include: ["ethernet-controller.yaml", "ethernet,fixed-link.yaml", "pinctrl-device.yaml"]
+
+properties:
+  interrupts:
+    required: true
+
+  nxp,interface-mode:
+    required: true
+    type: string
+    enum:
+      - "mii"
+      - "rmii"
+    description: |
+      The interface that the PHY is connected to the ENET
+
+  nxp,fixed-link:
+    type: boolean
+    description: |
+      The ethernet instance operates with a fixed link phy.
```

Comment:
Same comment as above.

Reply:
I don't know which comment you meant, but this property is not even used, I will just remove it, it's there by mistake I think, because there is now a fixed-link phy binding